### PR TITLE
task(CI): Make smoke test URLs configurable

### PR DESCRIPTION
**Pull request by dschom**
## Because

- We want to try running smoke tests against a new environment

## This pull request

- Adds pipeline parameters for configuring standard domains
- Transfers pipeline parameters to a set of standard envs
- Updates playwright tests to use these envs
- Updates functional content server tests to use these envs

## Issue that this pull request solves

Closes: FXA-7017

## Checklist

_Put an `x` in the boxes that apply_

- [X] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -279,6 +293,10 @@ commands:
           environment:
             CIRCLE_NODE_INDEX: << parameters.index >>
             CIRCLE_NODE_TOTAL: << parameters.total >>
+            FXA_ACCOUNTS_DOMAIN: << pipeline.parameters.accounts-domain >>
```
Comment (vbudhram) [inline .circleci/config.yml:296]: I probably just strip out the `FXA_` prefix.

**Review by vbudhram — COMMENTED** [review]
@dschom These changes make sense to me!

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,7 +28,21 @@ parameters:
   force-deploy-fxa-ci-images:
     type: boolean
     default: false
-
+  accounts-domain:
+    type: string
+    default: accounts.firefox.com
```
Comment (dschom) [inline .circleci/config.yml:33]: @jbuck Should we default to GCP urls now?

Comment (jbuck) [reply .circleci/config.yml:33]: I think keeping it to the AWS prod URLs is fine - I don't think we can change the stage URLs easily, think about how many clients have them

Comment (jbuck) [reply .circleci/config.yml:33]: Hm, we should change this url to the stage URL or change the herokuapp domains to the prod versions

**Review by jbuck — APPROVED** [review]
lgtm with two nits

**Review by vbudhram — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (dschom) [reply .circleci/config.yml]: I just looked at this again. I think we can leave them undefined, and then specify if/when we need to test a new URL. If these are pipeline parameters are not defined, the jobs will default to their typical settings, e.g. the defaults set in `functional-tests/lib/production.ts` and `functional-tests/lib/stage.ts`.

**Pull request merged**